rulesets/resources/301ruleset.rb

```diff
@@ -1686,7 +1686,14 @@ def self.set_systems_water_heater_reference(orig_bldg, new_bldg, is_all_electric
 
       heating_capacity = Waterheater.get_default_heating_capacity(fuel_type, @nbeds, orig_bldg.water_heating_systems.size) * 1000.0 # Btuh
 
-      location = orig_water_heater.location
+      #If 2022, reference WH is in default location, regardless of rated home location
+      if Constants.ERIVersions.index(@eri_version) >= Constants.ERIVersions.index('2019ABCD')
+        climate_zone_iecc = orig_bldg.climate_and_risk_zones.climate_zone_ieccs.select { |z| z.year == 2006 }[0]
+        location = Waterheater.get_default_location(orig_bldg, climate_zone_iecc)
```

Comment:
@jmaguire1 Note that 2019ABCD would be included here since you used `>=`. Probably this logic should be >= 2022 instead of >= 2019ABCD.

Also, what you have might be perfectly fine, but you could also just do `location = nil` in the if statement, such that the water heater location will be defaulted downstream when OS-HPXML processes each ERI building.

Reply:
Thanks for taking a look Scott! Getting caught up after my trip but I'll try to have this marked for review by our next meeting at the latest.